src/web3mail/sendEmail.ts

```diff
@@ -22,7 +22,7 @@ import {
   SendEmailResponse,
   SubgraphConsumer,
 } from './types.js';
-
+import * as ipfs from './../utils/ipfs-service.js';
```

Comment:
newline after imports

```suggestion
import * as ipfs from './../utils/ipfs-service.js';

```